### ISSUE TITLE
Missing l10n warnings on the log + Errors/issues...

### ISSUE THREAD

**Issue by FirenzeIT** [loki79uk/FS22_BulkFill#4]
Hello, the mod present warnings about l10n translation, it's possible to fix them please??

![Screenshot (57)](https://github.com/loki79uk/FS22_BulkFill/assets/113168067/69946b8a-f2b7-4e54-ab60-211b73b7716c)

**Comment by FirenzeIT** [comment]
Hello @loki79uk, i send you my modified l10n for italian language (on moddesc), which i hope to update this (and future) mod BulkFill, it's most useful. Thank you. [open with Notepad++]

[modDesc.txt](https://github.com/loki79uk/FS22_BulkFill/files/14099067/modDesc.txt)

**Comment by loki79uk** [comment]
Thank you, I will add it.

Is `Riemp In Blocco` acceptable?

GIANTS have rules that all words of titles must be capitalised, and I'm not sure about punctuation, so probably safer without.

**Comment by FirenzeIT** [comment]
Yes, Riemp. it's a shorten name with dot, the full name is Riempimento, so it's possible to add also this for have full lenght name and understand.. Full name is: Riempimento in Blocco, the rest it's the same as log... Ok, so for safe, use Riempimento in Blocco.

**Comment by FirenzeIT** [comment]
Good morning @loki79uk, i have also various errors with this mod, when i have 2 o 3 mods or 0 or 100.. It's possible to fix them please??:

C:/Users/User/Documents/My Games/FarmingSimulator2022/mods/FS22_BulkFill/BulkFill.lua:460: attempt to index a nil value
2024-02-11 11:10 Error: Running LUA method 'update'.
C:/Users/User/Documents/My Games/FarmingSimulator2022/mods/FS22_BulkFill/BulkFill.lua:460: attempt to index a nil value
2024-02-11 11:10 Error: Running LUA method 'update'.
C:/Users/User/Documents/My Games/FarmingSimulator2022/mods/FS22_BulkFill/BulkFill.lua:460: attempt to index a nil value
2024-02-11 11:10 Error: Running LUA method 'update'.
C:/Users/User/Documents/My Games/FarmingSimulator2022/mods/FS22_BulkFill/BulkFill.lua:460: attempt to index a nil value
2024-02-11 11:10 Error: Running LUA method 'update'.
C:/Users/User/Documents/My Games/FarmingSimulator2022/mods/FS22_BulkFill/BulkFill.lua:460: attempt to index a nil value

[log.txt](https://github.com/loki79uk/FS22_BulkFill/files/14232160/log.txt)

**Comment by loki79uk** [comment]
I realise now this should have been a new issue - I'll create one and comment there.

**Comment by FirenzeIT** [comment]
> I realise now this should have been a new issue - I'll create one and comment there.

Hi 🖐🏻 yes, the problem/error is due while I was refilling a 10000 lt seeder (original by default) with both seed and fertilizer pallets.. With your new update, I have tried to completely refill 2 same seeders, and so far I have not received any kind of problem/error.. Only the missing of the Italian language l10n..

**Comment by FirenzeIT** [comment]
Hi @loki79uk, i rewrite italian's language missing for implement in next update, if possible 👍🏻 

[modDesc.txt](https://github.com/loki79uk/FS22_BulkFill/files/14311772/modDesc.txt)

**Comment by loki79uk** [comment]
Thanks I will add it.  I need to change the design to use a separate file for each language like:
https://github.com/loki79uk/FS22_UniversalAutoload/tree/main/language

BUT if I create those files too soon, it is more work to add a new parameter into every file.

I don't know why the release version of FS still displays these missing messages, instead of just using the English as a default when a string is missing..

**Comment by FirenzeIT** [comment]
Hi, i don't know why.. Ok, take your time.. 🙃

Il Ven 16 Feb 2024, 15:02 Loki_79 ***@***.***> ha scritto:

**Comment by FirenzeIT** [comment]
Hi @loki79uk, can you also convert this mod to FS 25 please? 🙂

**Comment by loki79uk** [comment]
> Hi @loki79uk, can you also convert this mod to FS 25 please? 🙂

Yes, it is on my list, but UAL is taking a lot of time at the moment.  I have done the icon already :D

**Comment by FirenzeIT** [comment]
> > Hi @loki79uk, can you also convert this mod to FS 25 please? 🙂
> 
> Yes, it is on my list, but UAL is taking a lot of time at the moment. I have done the icon already :D

Ok, thank you for reply.
